github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -160,6 +160,38 @@ presubmits:
         resources:
           requests:
             memory: "29Gi"
+  - name: pull-kubevirt-e2e-k8s-multus-1.15.1
```

Comment:
Can we just replace `pull-kubevirt-e2e-k8s-multus-1.13.3`?

Reply:
I removed the test lane from the PR.

just leave the kubevirtci build and push jobs.

after we merge this I will create a new one for the kubevirt repo jobs to switch between the 1.13.3 to 1.15.1